lib/services/insights/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "azure-insights",
+  "author": "Microsoft Corporation",
+  "contributors": [
+    "Kamath, Prarthana <pkamath@microsoft.com>",
+    "Vannelli, Leo <leovann@microsoft.com>",
+    "Shen, Andy <andyshen@microsoft.com>",
+    "Ruela, Vinicius <viruela@microsoft.com>",
+    "Moussa, George <georgenm@microsoft.com>",
+    "McKinlay, David <davmc@microsofto.com>",
+    "Kulkarni, Jagadish <jagadisk@microsoft.com>",
+    "Calderon-Meza, Guillermo <gucalder@microsoft.com>"
+  ],
+  "version": "0.0.1",
```

Comment:
could you start with 0.9.0 unless you have specific reason?
0.0.1 is a valid version, but we want to to consistent with net-for-sdk.